.gitignore

```diff
@@ -7,6 +7,7 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+/.idea/dictionaires/talesra.xml
```

Comment:
I don't think we should have per-user details in .gitignore. @BenHenning?

Reply:
Solved from code side. Removed talesra.xml and introduced a common project-level dictionary